src/GitHub.Exports/Models/SimpleRepositoryModel.cs

```diff
@@ -18,6 +21,31 @@ public SimpleRepositoryModel(string name, UriString cloneUrl, string localPath =
             Icon = Octicon.repo;
         }
 
+        public SimpleRepositoryModel(string path)
+        {
+            if (path == null)
+                throw new ArgumentNullException("path");
+            if (!Directory.Exists(path))
+                throw new ArgumentException("Path does not exist", path);
+            var uri = VisualStudio.Services.GetRepoFromPath(path)?.GetUri();
+            var name = uri?.NameWithOwner;
+            if (name == null)
+                name = Path.GetDirectoryName(name);
```

Comment:
Huh? If `name` is null, then `Path.GetDirectoryName(name)` will also return null.

Did you mean `if (name != null)` perhaps? Or `if (!string.IsNullOrEmpty(name))`?